modules/react/avatar/lib/Avatar.tsx

```diff
@@ -35,6 +36,12 @@ export interface AvatarProps extends React.ButtonHTMLAttributes<HTMLButtonElemen
    * Will render an `div` tag instead of a `button` when defined.
    */
   as?: 'div';
+  /**
+   * The object-fit CSS property sets how the content of a replaced element,
+   * such as an <img> or <video>, should be resized to fit its container.
```

Comment:
We can maybe add something like `If you image is not a square, you can use this property to ensure the image is rendered properly`.

Reply:
Sure!